src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```
@@ -10,6 +10,13 @@
     <Features>$(Features.Replace('nullablePublicOnly', '')</Features>
   </PropertyGroup>
 
+  <!-- 
+    Add special trait to run local timezone validation test on devices.  We want to make sure the local timezone is correct and not UTC
```

I think this is clearer:

```suggestion
    Add special trait for running local timezone validation tests on certain devices e.g. in CI to make sure the local timezone is correct and not UTC. Disable these tests otherwise.
```